app/src/main/java/com/github/houseorganizer/houseorganizer/util/Util.java

```diff
@@ -20,4 +34,49 @@ public static boolean putEventStringsInData(Map<String, String> event, Map<Strin
         }
         return false;
     }
+
+    //<-------------------¦ Preferences ¦--------------------------------------------------->
+
+    public static final String SHARED_PREFS = "com.github.houseorganizer.houseorganizer.sharedPrefs";
+
+    public static SharedPreferences getSharedPrefs(Activity a) {
+        return a.getSharedPreferences(SHARED_PREFS, MODE_PRIVATE);
+    }
+
+    public static SharedPreferences.Editor getSharedPrefsEditor(Activity a) {
+        SharedPreferences prefs = getSharedPrefs(a);
+        return prefs.edit();
+    }
+
+    //<-------------------¦ Messages ¦--------------------------------------------------->
+
+    public enum ErrorType {
+        INVALID_PASSWORD,
+        INVALID_EMAIL,
+        EMAIL_USED,
+        INPUTS_EMPTY
+    }
+
+    public static void logAndToast(List<String> logTagAndMsg, Exception e, Context cx, String toastMsg) {
+        assert logTagAndMsg.size() == 2;
+
+        Log.w(logTagAndMsg.get(0), logTagAndMsg.get(1), e);
+        Toast.makeText(cx, toastMsg, Toast.LENGTH_SHORT).show();
+    }
```

Comment:
We can probably use the same message for the log and the toast, also I dont think putting the log tag and log message in a list makes sense other than for making codeclimate happy, we should talk about that and maybe ignore codeclimate
(or using the same message would also fix it)

Reply:
I don't think so, log is for the debugger and toast is for the user hence they cannot have the same message. 
We can talk about which type to use on our meeting.

Reply:
Okay I separated the list into two strings. I increased code climate limit on args because it made calendar methods fail.